### PR TITLE
Add gpt 345M and 13B example

### PR DESCRIPTION
**Pull request by tocean**

### CODE REVIEW

File: gpt3/pretrain_345m.sh [hunk 1]
```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
```
Comment (cp5555) [inline gpt3/pretrain_345m.sh:1]: Would you please add Microsoft trademark?

File: gpt3/README.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+Run the following command to train 345M GPT3 using fp16:
+```bash
+bash pretrain_345m.sh fp16
+```
+
+## Pretrain GPT3-345m with MS-AMP
+Run the following command to train 345M GPT3 using MS-AMP:
+```
+bash pretrain_345m.sh msamp
+
```
Comment (cp5555) [inline gpt3/README.md:36]: Remove empty line.

**Review by cp5555 — APPROVED** [review]

**Review by wkcn — APPROVED** [review]
LGTM. Thanks!

File: gpt3/pretrain_345m.sh (hunk 1 above)
Comment (tocean) [reply gpt3/pretrain_345m.sh:1]: Done.

File: gpt3/README.md (hunk 2 above)
Comment (tocean) [reply gpt3/README.md]: Done.

**Pull request merged**